project/partners/views.py

```diff
@@ -1,3 +1,11 @@
 from django.shortcuts import render
+from django.views import View
+from .models import Partner
 
-# Create your views here.
+class PartnersView(View):
+
+    template = '../templates/partners/sponsors.html'
+
+    def get(self, request, *args, **kwargs):
+        PartnerList = Partner.objects.all()
```

Comment:
Prefer lowercase for variable names:
`PartnerList` -> `partner_list` or `partners`
`{'Partners': ...}` -> `{'partners': ...}`